eng/Version.Details.xml

```diff
@@ -389,6 +389,10 @@
       <Uri>https://github.com/dotnet/core-setup</Uri>
       <Sha>094b6111d3d8c90d1111db4d73fd9faa755d9413</Sha>
     </Dependency>
+    <Dependency Name="runtime.win-x64.Microsoft.NETCore.DotNetAppHost" Version="3.0.0-preview7-27812-08" CoherentParentDependency="Microsoft.EntityFrameworkCore">
```

Comment:
Shouldn't the `CoherentParentDependency` be "Microsoft.Extensions.Logging"

Reply:
yeah bad copy pasta.